pkg/agent/proxy/proxier.go

```diff
@@ -613,6 +630,31 @@ func (p *proxier) installServices() {
 	}
 }
 
+func (p *proxier) getMCSExportedServiceClusterIP(serviceCIDRIPv4 *net.IPNet, endpointIP string) *agenttypes.ServiceGroupInfo {
+	var mcsLocalService *agenttypes.ServiceGroupInfo
+	// When the Endpoint is a local ClusterIP, it means the corresponding local Service is a member of the Multi-cluster Service.
+	if serviceCIDRIPv4 != nil && serviceCIDRIPv4.Contains(net.ParseIP(endpointIP)) {
+		mcsLocalService = &agenttypes.ServiceGroupInfo{
+			ClusterIP: net.ParseIP(endpointIP),
+		}
+	}
+	return mcsLocalService
+}
+
+func (p *proxier) getMCSExportedServiceGroupID(mcsLocalService *agenttypes.ServiceGroupInfo, svcPortName k8sproxy.ServicePortName) {
+	// mcWebhookServiceName "antrea-mc-webhook-service" is a pre-defined webhook Service for Antrea Multi-cluster
+	// instead of a Multi-cluster Service, so it should be excluded. For any Multi-cluster Service, its name will
+	// be a combination with prefix `antrea-mc-` and exported Service's name. So we need to remove the prefix to
+	// look up the exported Service.
+	if mcsLocalService != nil && svcPortName.Name != mcWebhookServiceName && strings.HasPrefix(svcPortName.Name, "antrea-mc-") {
```

Comment:
I think `mcsLocalService != nil` should be checked by the caller.

Reply:
Done